api/apps/articles/routes.coffee

```diff
@@ -1,5 +1,5 @@
 _ = require 'underscore'
-{ present, presentCollection } = Article = require './model'
+{ present, presentCollection } = Article = require './model/index.coffee'
```

Comment:
Pro tip, Node recognizes `index` as a default so you could leave this as `require './model'` and it would work the same.